project/Dependencies.scala

```diff
@@ -30,6 +30,7 @@ object Dependencies {
   val awsS3 = "com.amazonaws" % "aws-java-sdk-s3" % awsVersion
   val awsDynamo = "com.amazonaws" % "aws-java-sdk-dynamodb" % awsVersion
   val awsLambdaCore = "com.amazonaws" % "aws-lambda-java-core" % "1.1.0"
+  val awsTranscoder = "com.amazonaws" % "aws-java-sdk-elastictranscoder" % awsVersion
```

Comment:
I believe the SDK for building lambdas operates on a different schedule to the AWS clients